lib/webdriverio.js

```diff
@@ -200,7 +200,7 @@ let WebdriverIO = function (args, modifier) {
         err.shotTaken = true
         return takeScreenshot(err)
             .catch((e) => logger.log('\tFailed to take screenshot on reject:', e))
-            .then(() => fail(err, onRejected))
+            .then(() => { throw err })
```

Comment:
The trow will bubble up and trigger the `reject` (line [#L173](https://github.com/wvankuipers/webdriverio/blob/61b43ed8f5cb868e41251bcdff54a432136b46f5/lib/webdriverio.js#L173)) again, triggering [#L190](https://github.com/wvankuipers/webdriverio/blob/61b43ed8f5cb868e41251bcdff54a432136b46f5/lib/webdriverio.js#L190) and [#L181](https://github.com/wvankuipers/webdriverio/blob/61b43ed8f5cb868e41251bcdff54a432136b46f5/lib/webdriverio.js#L181).